Rakefile

```
@@ -36,3 +36,32 @@ task default: %i[
   test
   internal_investigation
 ]
+
+desc 'Generate a new cop template'
+task :new_cop, [:cop] do |_task, args|
+  require 'rubocop'
+
+  cop_name = args.fetch(:cop) do
+    warn 'usage: bundle exec rake new_cop[Department/Name]'
+    exit!
+  end
+
+  github_user = `git config github.user`.chop
+  github_user = 'your_id' if github_user.empty?
+
+  generator = RuboCop::Cop::Generator.new(cop_name, github_user)
+
+  generator.write_source
+  generator.write_spec
+  generator.inject_require(root_file_path: 'lib/rubocop/cop/minitest_cops.rb')
+  generator.inject_config(config_file_path: 'config/default.yml',
+                          version_added: bump_minor_version)
```

RuboCop Minitest's max of `Layout/LineLength` is `120`.
https://github.com/rubocop-hq/rubocop-minitest/blob/v0.9.0/.rubocop.yml#L50-L51

One line will be enough.

```suggestion
  generator.inject_config(config_file_path: 'config/default.yml', version_added: bump_minor_version)
```